### ISSUE TITLE
About pixel shuffle

### ISSUE THREAD

**Issue by P66094125** [MagicLeapResearch/SuperPointPretrainedNetwork#24]
Do you use the pixel shuffle in the superpoint model? if not, can you try to reinterpret the following sentence's meaning ? 

In  sentence in the superpoint paper:
Unfortunately, upsampling layers tend to add a high amount of computation and can introduce unwanted checkerboard artifacts, thus we designed the interest point detection head with an explicit decoder1 to reduce the computation of the model. This decoder has no parameters and is known as “sub-pixel convolution” or “depth to space” in TensorFlow or “pixel shuffle” in PyTorch.

thank you.
